### ISSUE TITLE
Conform to WCAG AA

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/dpul-collections#490]
### Summary or User Story

Assess all pages of the site using browser plugins for adherence to WCAG AA accessibility requirements. Create and prioritize tickets that reflect work required to conform.

### Acceptance Criteria

- [ ] All accessibility violations are ticketed

### First step

run lighthouse or wave
